ghc-lib-gen/src/Ghclibgen.hs

```diff
@@ -87,7 +87,8 @@ ghcLibParserIncludeDirs _forDepends ghcFlavor =
         series | series >= GHC_9_10 ->
                  [ "rts/include"
                  , "rts/include/stg"
-#if __GLASGOW_HASKELL__ == 910 || __GLASGOW_HASKELL__ == 908
+#if __GLASGOW_HASKELL__ == 910
```

Comment:
Is there a reason we're dropping 908 here?

Reply:
good catch. have pushed again with "the right" fix. it's busy work. the hackage linter won't allow us to upload a package with an empty directory so this has to be guarded against. it used to allow this but it became stricter.

Reply:
:+1: